creativecloud/features/firefly/firefly-interactive.js

```diff
@@ -4,8 +4,23 @@ const miloLibs = setLibs('/libs');
 const { createTag } = await import(`${miloLibs}/utils/utils.js`);
 const { createSelectorTray, createEnticement, createPromptField } = await import('../interactive-elements/interactive-elements.js');
 
-let interactiveElemsText;
 let media;
+let mediaP;
+
+function focusOnInput() {
+  media.querySelector('#promptinput')?.focus();
+}
+
+function waitForGeoModal() {
```

Comment:
Added waitForGeoModal() function to handle the focus on prompt input when georouting pop is open and then closed. However this function only handles the use when user clicks the X button on the modal. And it will not work if the modal is closed by clicking on the page. I'll look for other ways to add focus to input field if georouting modal pops up.

Reply:
Updated code to use css for adding blinking cursor and removed this javascript function